tcms/testplans/static/testplans/js/get.js

```diff
@@ -311,6 +313,39 @@ function toolbarEvents(testPlanId) {
         sortTestCases();
     });
 
+
+    // always initialize the sortable list however you can only
+    // move items using the handle icon on the left which becomes
+    // visible only when the manual sorting button is clicked
+    const manualSortList = new Sortable($("#testcases-list")[0], {
+        animation: 150,
+        handle: '.handle',
+        dataIdAttr: 'data-testcase-pk',
+    });
+
+    // IMPORTANT: this is not empty b/c Sortable() is initialized *after*
+    // all of the test cases have been rendered !!!
+    const initialOrder = manualSortList.toArray();
+
+    $('.js-toolbar-manual-sort').click(function(event) {
+        $('.js-toolbar-manual-sort').find('span').toggleClass('fa-sort');
+        $('.js-toolbar-manual-sort').find('span').toggleClass('fa-check-square');
```

Comment:
You can pass array of class names to toggleClass

Reply:
Also if we add $(this).blur(), we will remove the focus from the manual-sort button for better ux